src/ucp/api/ucp.h

```diff
@@ -1293,6 +1293,31 @@ ucs_status_t ucp_mem_map(ucp_context_h context, ucp_mem_map_params_t *params,
 ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh);
 
 
+enum ucp_mem_advice_flags {
```

Comment:
but the enum holds 'advices'. Thats why it is called advices_flags or perhaps i should rename it to advice_values